### ISSUE TITLE
[TESTING]: OMB Info - Unit tests MUST run axe checks with the modal open for better code coverage

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/vets-design-system-documentation#224]
@1Copenut commented on [Wed Jun 03 2020](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9816)

## [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The OMB Info component includes a modal window of legal text. The unit tests don't run an axe check on this modal. The component depends on this modal, so it must include an axe check with the modal open. This looks like it can be passed as a prop easily in the `OMBInfo.unit.spec.jsx` file.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Unit test file includes an axe check with the modal window open
- [ ] Axe checks return 0 violations

## Environment

* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/ombinfo/

**Comment by ddzz** [comment]
This component is failing an aXe scan when the modal is open:
```
	Error: [serious] ARIA dialog and alertdialog nodes must have an accessible name
	Id: aria-dialog-name
	See https://dequeuniversity.com/rules/axe/4.1/aria-dialog-name?application=axeAPI

	<div class="va-modal va-modal-large" id="omb-modal" role="alertdialog" aria-modal="true">
	#omb-modal
```

**Comment by 1Copenut** [comment]
@ddzz The OMB Modal needs a `titleId` prop passed to give it an accessible label. I traced it back through our Modal component:

* https://github.com/department-of-veterans-affairs/component-library/blob/master/src/components/Modal/Modal.jsx#L156

```diff
render() {
    const { resBurden, ombNumber, expDate } = this.props;
+   const titleIdString = "OMB-meaningful-string"

    return (
      <div className="omb-info">
        {resBurden && (
          <div>
            Respondent burden: <strong>{resBurden} minutes</strong>
          </div>
        )}
        {ombNumber && (
          <div>
            OMB Control #: <strong>{ombNumber}</strong>
          </div>
        )}
        <div>
          Expiration date: <strong>{expDate}</strong>
        </div>
        <div>
          <button className="va-button-link" onClick={this.openModal}>
            Privacy Act Statement
          </button>
        </div>
        <Modal
          cssClass="va-modal-large"
          contents={
            this.props.children
              ? this.props.children
              : this.modalContents(resBurden)
          }
          id="omb-modal"
          visible={this.state.modalOpen}
          onClose={this.closeModal}
+         titleId={titleIdString}
        />
      </div>
    );
```
